### PR TITLE
ADAPT-4215: Port Vault variable management strategy.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Ports vault variable management strategy from give.stanford.edu
- Sets up branch deploy website

# Review By (Date)
- Whenever

# Criticality
- Low

# Review Tasks

## Setup tasks and/or behavior to test

1. Review code
2. Review preview build website for irregularities
3. Check out branch and follow installation instructions to get a local site up and running
4. Review vault.stanford.edu and the giving secrets for any irregularities
5. Review readme instructions
6. Run installation steps and validate you get a running local environment

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
> 

😂  I will review this now, @sherakama

File: README.md [hunk 1]
```diff
@@ -12,65 +12,107 @@ Description
...
-cp example.env.development .env.development
-* Manually add the access token to .env.development *
-nvm use
-npm install
-gatsby develop
+cp example.env .env
+* Manually add the vault secret and role id to .env
+* You can get the variables by running `netlify login` then `netlify link` and then `netlify env`
+* After the VAULT_ROLE_ID and VAULT_SECRET_ID environment variables have been set run
+npm run vault
```
Comment (yvonnetangsu) [inline README.md:24]:
```suggestion
npm install
npm run vault
```

File: README.md [hunk 2]
```diff
@@ -12,65 +12,107 @@ Description
...
+* Then fire up your development server using gatsby
+npm run dev
```
Comment (yvonnetangsu) [inline README.md:26]:
```suggestion
npm run develop
```
Looks like Giving doesn't have `npm run dev` - please feel free to change `npm run develop` to `npm run dev` in the package.json file, @sherakama 😄  Less typing = good

**Review by yvonnetangsu — APPROVED** [review]
Preapproving with a couple tiny edits - everything working for me! Thanks for doing this, @sherakama 🎉 😄

**Review by rebeccahongsf — APPROVED** [review]
Aside side from the documentation and/or script adjustments that Yvonne mentioned, everything looks good to me! 🚀

**Pull request merged**